packages/lodestar/test/unit/sync/backfill/verify.test.ts

```diff
@@ -0,0 +1,51 @@
+import {BackfillSyncErrorCode} from "./../../../../src/sync/backfill/errors";
+import {Json} from "@chainsafe/ssz";
+import {createIBeaconConfig} from "@chainsafe/lodestar-config";
+import {config} from "@chainsafe/lodestar-config/default";
+import {phase0, ssz} from "@chainsafe/lodestar-types";
+import {expect} from "chai";
+import {readFileSync} from "fs";
+import {verifyBlockSequence} from "../../../../src/sync/backfill/verify";
+import path from "path";
+
+describe("backfill sync - verify block sequence", function () {
+  //mainnet validators root
+  const beaconConfig = createIBeaconConfig(
+    config,
+    ssz.Root.fromJson("0x4b363db94e286120d76eb905340fdd4e54bfe9f06bf33ff6cf5ad27f511bfe95")
+  );
+
+  it("should verify valid chain of blocks", function () {
+    const blocks = getBlocks();
```

Comment:
Can we read the file and parse it only once for the describe block? You can either cache the return or call getBlocks in a before() block

Reply:
It takes lik 10ms to do that so I think we are fine, skipped caching since tests and code is modifying original array so we would have to copy array of blocks between tests